### DISCUSSION TITLE
Queues in Adonis 5

### DISCUSSION THREAD

**Discussion by akraminakib** [adonisjs/core#1638] [General]
Hey guys,

I'm curious to know if there currently is or will be any first class support for queues in Adonis 5?

I see it being mentioned on the [blog](https://preview.adonisjs.com/blog/introducing-adonisjs-v5/#new-functionality-to-be-added) but nothing in the roadmap Trello board for it.

**Comment by thetutlage** [comment]
@akraminakib. There will be, but it is not prioritized yet. I will most probably go with [bull](https://github.com/OptimalBits/bull)

**Reply by RodolfoSilva** [reply]
[bullmq](https://github.com/taskforcesh/bullmq) 😜

**Reply by reg2005** [reply]
@RodolfoSilva It's great! BullMQ is best queue. I made [simple provider for it](https://github.com/reg2005/adonis5-bullmq).

**Reply by RodolfoSilva** [reply]
Great, good work! I'll look deep latter.
 
I've rewrite the @Rocketseat library to be compatible with Adonis 5, take a look (https://github.com/rocketseat/adonis-bull ). Right now is in alpha.

But, is good to know about another implementation :D  Thanks.

**Reply by reg2005** [reply]
@RodolfoSilva Your alpha looks amazing! I will use it on the week. Thx

**Comment by akraminakib** [comment]
@thetutlage Awesome thanks!

**Comment by detygon** [comment]
@thetutlage Is there any blog post or resource I can refer to if I want to use it now ?
